src/etc/inc/auth.inc

```diff
@@ -360,6 +360,13 @@ function local_backed($username, $passwd) {
 		return false;
 	}
 
+	if ($user['bcrypt-hash']) {
+		if (password_verify($passwd, $user['bcrypt-hash'])) {
```

Comment:
OK, just leave it as is for now and we can revisit and change all of them at once